hunting/linux/queries/defense_evasion_via_hidden_process_execution.toml

```
@@ -0,0 +1,30 @@
+[hunt]
+author = "Elastic"
+description = """
+This hunt identifies hidden process executions on Linux systems. It detects processes executed from hidden files, which are often used by malicious actors to conceal their activities. By focusing on hidden files rather than directories, this hunt aims to catch stealthy processes while minimizing noise.
+"""
+integration = ["endpoint"]
+uuid = "00461198-9a2d-4823-b4cc-f3d1b5c17935"
+name = "Hidden Process Execution"
+language = ["ES|QL"]
+license = "Elastic License v2"
+notes = [
+    "Included only hidden files, excluding hidden directories, as hidden directories are common in Unix.",
+    "Included a process or parent process count of <= 3, and a host count of <= 3 to eliminate common processes across different hosts.",
+]
+mitre = ["T1036.004", "T1059"]
+
+query = [
+'''
+from logs-endpoint.events.process-*
+| where @timestamp > now() - 180 day
+| where host.os.type == "linux" and event.type == "start" and event.action == "exec" and (
+  (process.executable rlike "/[^/]+/\\.[^/]+")
+)
```

unnecessary parenthesis ? 

```suggestion
| where host.os.type == "linux" and event.type == "start" and event.action == "exec" and 
  process.executable rlike "/[^/]+/\\.[^/]+"
```